apps/wallet/src/ui/index.tsx

```diff
@@ -69,15 +69,18 @@ function AppWrapper() {
                          * making the API provider instance a reactive value and moving it out of the redux-thunk middleware
                          */}
                         <Fragment key={network}>
-                            <QueryClientProvider client={queryClient}>
+                            <PersistQueryClientProvider
```

Comment:
are there any instances in the wallet where we need to invalidate the persisted cache?

Reply:
Verified that the cached queries and mutations get reset on network switches.

Reply:
Also added a cache reset on logout